testsuite/gateways/wasm/extension.py

```diff
@@ -51,10 +52,23 @@ def ingress_url(self):
             route = self.mesh.routes["istio-ingressgateway"]
             if "tls" in route["spec"]:
                 self._ingress_url = "https://" + route["spec"]["host"]
```

Comment:
`return "https://" + route["spec"]["host"]`

Reply:
This would cause the variable `self._ingress_url` to be useless as there is no assignment to it.